route53_static.tf

```diff
@@ -21,12 +21,37 @@ resource "aws_route53_record" "root_CAA" {
 # Generation of the domain identity token and DKIM keys in SES.
 # ------------------------------------------------------------------------------
 
+# We should consider upgrading to aws_sesv2_email_identity, although
+# it would be a destructive upgrade:
+# https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity
+#
+# See also
+# https://github.com/trussworks/terraform-aws-ses-domain/blob/main/main.tf
```

Comment:
Can you please create an issue for this so that we don't forget about it?

Reply:
Please see issue #109 and commit a97a68f.

Reply:
Excellent, thanks!